app/controllers/global_issue_templates_controller.rb

```diff
@@ -5,8 +5,8 @@ class GlobalIssueTemplatesController < ApplicationController
   helper :issues
   include IssuesHelper
   menu_item :issues
-  before_filter :find_object, :only => [ :show, :edit, :destroy ]
-  before_filter :require_admin, :find_user, :only => [ :index, :new, :show ], :except => [ :preview ]
+  before_filter :find_object, only: [ :show, :edit, :destroy ]
+  before_filter :require_admin, :find_user, only: [ :index, :new, :show ], excep: [ :preview ]
```

Comment:
__[rubocop]__
* Space inside square brackets detected. [:ref](https://github.com/bbatsov/ruby-style-guide#no-spaces-braces)
* Line is too long. [94/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)